tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php

```diff
@@ -214,17 +214,16 @@ protected function createResponseMock($statusCode, $reasonPhrase, $body)
         if (is_array($body)) {
             $response
             ->expects($this->once())
-            ->method('json')
-            ->with()
-            ->will($this->returnValue($body));
+            ->method('getBody')
+            ->with(true)
+            ->will($this->returnValue($json));
         } else {
-            $exception = new \Guzzle\Common\Exception\RuntimeException();
-
+            // Error response
             $response
             ->expects($this->once())
-            ->method('json')
-            ->with()
-            ->will($this->throwException($exception));
+            ->method('getBody')
+            ->with(true)
+            ->will($this->returnValue($body));
```

Comment:
[$json     = is_array($body) ? json_encode($body) : $body;](https://github.com/tmatsuo/php-coveralls/blob/circle/tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php#L197)

and (then)[https://github.com/tmatsuo/php-coveralls/blob/circle/tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php#L214]:
```php
if (is_array($body)) {
    // use $json
} else {
    // use $body, which is equals to $json
}
```